account_banking_pain_base/migrations/11.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Brain-Tec Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def set_account_holder(cr):
```

Comment:
Don't duplicate code. Import post_init_hook method. Any way, I also agree this shouldn't be filled by default.

Reply:
I think the best way is first to rename the post_install file to hooks which is the name it should have following the guidelines. Then, as you said, import and use the method in the post-migration file.

Reply:
But as said, you shouldn't fill the same data, so remove all. My comment is only "didactic"